tools/flang2/flang2exe/bih.h

```diff
@@ -63,6 +63,7 @@ typedef struct {
       unsigned parsect : 1;   /* bih belongs to a parallel section */
       unsigned ujres : 1;     /* bih contains ujresidual start & count info */
       unsigned simd : 1;       /* bih contains simd code */
+      unsigned nosimd : 1;       /* bih contains simd code */
```

Comment:
Should the comment say `bih does not contain simd code`?

Reply:
Done.